CMakeLists.txt

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-cmake_minimum_required(VERSION 3.5)
+cmake_minimum_required(VERSION 3.5.0...3.31.3)
```

Comment:
From CMake docs:

> If the running version of CMake is older than 3.12, the extra ... dots will be seen as version component separators, resulting in the ...<max> part being ignored and preserving the pre-3.12 behavior of basing policies on <min>.

Shouldn't we add some comment (at least to the root CMake file) to reflect that?

Reply:
I added a comment with text from the documentation.